# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentChild.h
# Commit: 9be5ae499d84
# Full Hash: 9be5ae499d84aca2d74d3d42dbe7b120987a1502
# Author: Dimi Lee <dlee@mozilla.com>
# Date: 2020-04-23 21:43:09
# Regressor Bug: 1616775
# File Overlap Count: 1
# Description:
#   Bug 1616775 - P6. Add OnAllowAccessFor to process tasks that need a parent window. r=timhuang,baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D71015
# ==============================================================================

diff -r 577f090dae54 -r 9be5ae499d84 dom/ipc/ContentChild.h
--- a/dom/ipc/ContentChild.h	Thu Apr 23 14:25:23 2020 +0000
+++ b/dom/ipc/ContentChild.h	Thu Apr 23 14:25:31 2020 +0000
@@ -673,6 +673,11 @@
       CrossProcessSHEntry* aOldEntry, CrossProcessSHEntry* aNewEntry,
       const MaybeDiscarded<BrowsingContext>& aContext);
 
+  mozilla::ipc::IPCResult RecvOnAllowAccessFor(
+      const MaybeDiscarded<BrowsingContext>& aContext,
+      const nsCString& aTrackingOrigin, uint32_t aCookieBehavior,
+      const ContentBlockingNotifier::StorageAccessGrantedReason& aReason);
+
 #ifdef NIGHTLY_BUILD
   // Fetch the current number of pending input events.
   //